web/package.json

```diff
@@ -30,7 +30,7 @@
     "jotai": "^2.6.4",
     "jsdom": "^24.0.0",
     "json5": "^2.2.3",
-    "next": "14.1.0",
```

Comment:
let's add an engines statement to make sure it can only be run with `node = 20.x`, and pnpm